WordPress/twentytwentyfive
# Services - Subscriber only section

Testing Instructions
Create a new page.
Add the pattern.
Confirm that it looks like the designs (spacing/colors/etc.).

Closes https://github.com/WordPress/twentytwentyfive/issues/77
[Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1808&node-type=SYMBOL&t=PTgblFBFp3QdQSCt-0)
